### PR TITLE
Enable decklink

### LINKED ISSUE #19: Blackmagic decklink support
It would be very helpful if --enable-decklink were a supported option in this tap! I may be missing something, but I do not see a way to build wit that option. BlackMagic requires the includes to be downloaded by the user, but an option to just point to those includes in the brew command line would be great.

### PR DESCRIPTION
**Pull request by retokromer**
should resolve https://github.com/varenc/homebrew-ffmpeg/issues/19

### CODE REVIEW

**Comment by retokromer** [comment]
cc @wblj

File: Formula/ffmpeg.rb [hunk 1]
```diff
@@ -146,6 +147,12 @@ def install
     args << "--enable-openssl" if build.with? "openssl"
 
     # packages that need additional license options
```
Comment (slhck) [inline Formula/ffmpeg.rb]: Actually, that comment is a little misplaced. We should group license-related code and build-related code.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ This formula features the following libraries:
 | `x265` | ✔ | x265 HEVC encoder |
 | `xz` | ✔ | XZ compression support |
 | `chromaprint` | | Chromaprint audio fingerprinting library |
+| `decklink`| | Enable DeckLink support |
```
Comment (slhck) [inline README.md]: Minor nit: space missing.

**Comment by slhck** [comment]
I don't have a DeckLink card with which I could test this. Otherwise looks good to me.

File: README.md (hunk 2 above)
Comment (retokromer) [reply README.md]: done, thanks!

File: Formula/ffmpeg.rb (hunk 1 above)
Comment (retokromer) [reply Formula/ffmpeg.rb]: indeed! updated

**Comment by slhck** [comment]
Thanks for the fixes! If we merge this, we could squash everything. I guess it does not affect the regular build anyway, so if you want to merge, go ahead. (Cannot test the DeckLink build at this stage.)

**Pull request merged**